packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -1,7 +1,13 @@
 /* eslint-disable valid-jsdoc */
-import {Event, LocalMicrophoneStream, RoapMediaConnection} from '@webex/internal-media-core';
+import {
+  Event,
+  LocalMicrophoneStream,
+  LocalStreamEventNames,
+  RoapMediaConnection,
+} from '@webex/internal-media-core';
 import {createMachine, interpret} from 'xstate';
 import {v4 as uuid} from 'uuid';
+import {LocalMicrophoneStreamEventNames} from '@webex/media-helpers';
```

Comment:
Can we remove this import? This has not been used anywhere

Reply:
Removed it